src/QirRuntime/lib/QIR/quantum__qis_internal.hpp

```diff
@@ -13,10 +13,17 @@
 #endif
 
 // For test purposes only:
-namespace Quantum { namespace Qis { namespace Internal  // Replace with `namespace Quantum::Qis::Internal` after migration to C++17.
+namespace Quantum // Replace with `namespace Quantum::Qis::Internal` after migration to C++17.
```

Comment:
Would you mind filing a bug for migrating to C++17? Hopefully, it should be as simple as changing 14 to 17 in `set(CMAKE_CXX_STANDARD 14)` in the root CMakeLists.txt.

Reply:
;-)
I tried, it cases a compiler error in (at least) one of the headers in MS VC++ (and that error is reported for dozens of translation units (.cpp files)).
Filed #523